src/Dealer.ts

```diff
@@ -186,7 +186,8 @@ export class Dealer {
         const originalPosition = updatedPositionResult.value.originalPosition
         const updatedPosition = updatedPositionResult.value.updatedPosition
 
-        exposureInUsd = updatedPosition.exposureInUsd
+        if (updatedPosition) exposureInUsd = updatedPosition.exposureInUsd
```

Comment:
you should always use `{ }` around `if / else` this can avoid bugs when doing auto merging.

Reply:
good catch thx & done!